samples/04_gis_analysts_data_scientists/classifying_human_activity_using _tabPFN_classifier.ipynb

```diff
@@ -0,0 +1,2168 @@
+{
```

Comment:
<p>use proper capitalization, change Mobile dataset to Mobile Dataset</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2189/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>done</p>